R/remote.R

```diff
@@ -1,5 +1,9 @@
 
-isChildProcess <- function() {
+#' Is this a job
```

Comment:
We should expand the documentation here. (Used to detect RStudio "jobs"; that is, R sessions that are run either explicitly as Jobs from the IDE, or certain sub-processes launched for the Build pane.)

Reply:
I'll look in the details. Can we then disambiguate between jobs and these "certain sub-processes" ... ?

Reply:
Hmm. Technically, this is true for any R process launched using our AsyncRProcess, but the ones relevant to users of rstudioapi are probably only Jobs and the Build pane. So maybe that's all we should mention in the documentation?